client/src/components/common/SprintModal/CreateSprintModal/index.tsx

```diff
@@ -82,6 +123,54 @@ const CreateSprintModal = (props: Props) => {
 							}
 						/>
 					</Form.Field>
+					<Form.Field
+						label="Duration"
+						control={() => (
+							<Dropdown
+								value={duration}
+								placeholder="Duration"
+								search
+								selection
+								options={DURATIONS}
+								onChange={handleDurationPick}
+							/>
+						)}
+					/>
+					<Form.Field
+						label="Start date"
+						error={!isDateValid}
+						control={() => (
+							<div style={{ display: 'flex', flexDirection: 'column' }}>
+								<DatePicker
+									name="StartTime"
+									dateFormat="MM/dd/yyyy h:mm aa"
+									showTimeSelect
+									selected={startDate}
+									onChange={handleStartDatePick}
+								/>
+								{!isDateValid && (
+									<span style={{ color: 'red' }}>
+										The start date of a sprint must be before the end date.
+									</span>
```

Comment:
Error is showed when start date is picked manually and is still before end date. Link to trello - [click me](https://trello.com/c/jZYlZHjG/263-add-start-and-end-date-for-the-sprint#comment-5f48ad0a6798f274e05f203e)